Content.Server/Morgue/Components/MorgueEntityStorageComponent.cs

```diff
@@ -93,7 +94,10 @@ protected override void OpenStorage()
                 _tray = Owner.EntityManager.SpawnEntity(_trayPrototypeId, Owner.Transform.Coordinates);
                 var trayComp = _tray.EnsureComponent<MorgueTrayComponent>();
                 trayComp.Morgue = Owner;
-                EntityQuery = new IntersectingEntityQuery(_tray);
+                var physicsManager = EntitySystem.Get<SharedBroadPhaseSystem>();
+                var box = new Box2(Owner.Transform.WorldPosition + new Vector2(-0.25f, -0.25f),
+                                   Owner.Transform.WorldPosition + new Vector2(0.25f, 0.25f));
```

Comment:
Get the world AABB from the physics component instead. Would be nice if you made a helper for this instead.